docs/ECLLanguageReference/ECLR_mods/BltInFunc-WHEN.xml

```diff
@@ -10,7 +10,7 @@
       </indexterm><indexterm>
         <primary>WHEN Function</primary>
       </indexterm>(</emphasis><emphasis>trigger, action</emphasis><emphasis
-    role="bold"> [ BEFORE | SUCCESS | FAILURE] )</emphasis></para>
+    role="bold"> [, BEFORE | SUCCESS | FAILURE] )</emphasis></para>
```

Comment:
WHEN(trigger, action [BEFORE | SUCCESS | FAILURE] ) is the current described syntax.
Since nothing follows the optional extras the ',' is also optional and thus goes inside the [ ]. The only thing I deliberated was the ws either side of '|', should this be there or should there be a ',' per possible option. I looked at other examples and the single ',' and ws surrounding '|' seemed to be the norm.